.gitattributes

```diff
@@ -4,3 +4,4 @@
 *.dx filter=lfs diff=lfs merge=lfs -text
 *.spe filter=lfs diff=lfs merge=lfs -text
 *.xrdml filter=lfs diff=lfs merge=lfs -text
+*.in filter=lfs diff=lfs merge=lfs -text
```

Comment:
Does this work @PeterKraus? I thought lfs was more magic than this (and GH normally clearly delinates LFS files in PRs) We can probably just amend the PR directly if not (no need for @edan-bainglass to worry)

Reply:
I cannot remember. But this github docs page seems to say it should be enough: https://docs.github.com/en/repositories/working-with-files/managing-large-files/configuring-git-large-file-storage

Reply:
It did not work -- after merging when I pull the repo I get a warning about files not being pointers. Will correct now.